### ISSUE TITLE
 Moving sandbox to folder - Cannot read property 'toJS' of undefined

### ISSUE THREAD

**Issue by rajjejosefsson** [codesandbox/codesandbox-client#2490]
# 🐛 bug report

## Description of the problem

Trying to move sandbox to a subfolder

```
helpers.js:90 Uncaught TypeError: Cannot read property 'toJS' of undefined
    at folder-drop-target.js:18
    at Object.drop (folder-drop-target.js:116)
    at t.drop (createTargetFactory.js:45)
    at drop.js:44
    at drop.js:26
    at Array.forEach (<anonymous>)
    at e.<anonymous> (drop.js:25)
    at Object.drop (DragDropManagerImpl.js:65)
    at e.value (HTML5Backend.js:642)
    at r (helpers.js:74)
```

seems that the prop "selectedSandboxes" is undefined when adding to subfolder, but to a parent folder works fine

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |
| Browser         | Chrome Version 76 and Firefox
| Operating System | OSX

**Comment by SaraVieira** [comment]
Hey!

Sorry about that, we know what that is and will be on that as soon as possible

**Comment by christianalfoni** [comment]
This has now been fixed and will be deployed later today, thanks so much for the feedback @rajjejosefsson. We have a few bumps after transitioning over to Typescript and new state management, but it is an investment to avoid these kinds of issues moving forward :)

**Comment by rajjejosefsson** [comment]
Okay just wanted to make sure it was addressed :) keep going guys! ;)

**Comment by MichaelDeBoey** [comment]
@CompuIves This one can be closed since #2491 is merged
